code/espurna/garland.cpp

```diff
@@ -87,11 +138,15 @@ constexpr size_t palsSize() { return sizeof(pals)/sizeof(pals[0]); }
 Adafruit_NeoPixel pixels = Adafruit_NeoPixel(GARLAND_LEDS, GARLAND_D_PIN, NEO_GRB + NEO_KHZ800);
 Scene scene(&pixels);
 
-Anim* anims[] = {new AnimStart(), new AnimPixieDust(), new AnimSparkr(), new AnimRun(), new AnimStars(), new AnimSpread(), 
-                 new AnimRandCyc(), new AnimFly(), new AnimComets(), new AnimAssemble(), new AnimDolphins(), new AnimSalut()};
+Anim* anims[] = {new AnimGlow(), new AnimStart(), new AnimPixieDust(), new AnimSparkr(), new AnimRun(), new AnimStars(), new AnimSpread(), 
+                 new AnimRandCyc(), new AnimFly(), new AnimComets(), new AnimAssemble(), new AnimDolphins(), new AnimSalut(), new AnimFountain()};
 
 constexpr size_t animsSize() { return sizeof(anims)/sizeof(anims[0]); }
 
+#define START_ANIMATION  1
+Anim* _currentAnim       = anims[1];
+Palette* _currentPalette = &pals[0];
+auto one_color_palette = std::unique_ptr<Palette>(new Palette("White", {0xffffff}));
```

Comment:
Check out https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique
I did a backport, since I constantly ran into this issue:
https://github.com/xoseperez/espurna/blob/7434ea3878f9a81752694fa87127cec676d58138/code/espurna/compat.h#L107-L110

Reply:
Not quite understand this tricky issue with creating unique_ptr. I unable to create unique_ptr with make_unique just placing arguments into it. Finally I found working set as this: 
`auto one_color_palette = std::make_unique<Palette>("White", std::vector<Color>({0xffffff}));`
Would it be better?

Reply:
s/this issue/make_unique not being available in the c++11/

initializer_list template deduction is very dumb though, i totally forgot. it either wants `std::vector<Color>` or `std::initializer_list<Color>`. make_unique should always be preferred... for multiple reasons, here being we only specify the type once
e.g.
```cpp
using Colors = std::vector<Color>; // also note other declarations, maybe also inside of namespace { ... }
auto one_color_palette = std::make_unique<Palette>("White", Colors{{0xffffff}});
```